README.md

```
@@ -527,6 +528,106 @@ Removed locks folder: /workspace/frappe-bench/sites/development.localhost/locks
+cwcli restore my-project --site production.localhost
+
+# Restore with verbose output for debugging
+cwcli restore my-project -v
+
+# Provide password via command line (not recommended for production)
+cwcli restore my-project --mariadb-root-password "secret123"
+```
+
+**Example Session:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)